pandas/io/formats/format.py

```diff
@@ -1520,17 +1520,26 @@ def __init__(
     def _format_strings(self) -> List[str]:
         """ we by definition have DO NOT have a TZ """
         values = self.values
+        flat_values = values.ravel()
 
-        if not isinstance(values, DatetimeIndex):
-            values = DatetimeIndex(values)
+        if not isinstance(flat_values, DatetimeIndex):
+            flat_values = DatetimeIndex(flat_values)
```

Comment:
`DatetimeIndex` appears to only support 1-d values. It will actually construct with dim > 1 but can error at some places, so pass flattened values just to be safe.

Reply:
could use DatetimeArray?

Reply:
Yeah, that is a little cleaner since `DatetimeArray._format_native_types` is the only thing being used here. It still might be good to call `values.ravel()` here since it's needed if `self.formatter` is set, although not really necessary otherwise.